### ISSUE TITLE
Fix accessibility labels on profile page

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1120]
Work in progress 

> For [accessibility] I’d recommend finding a screen reader user who is also a developer to help build out requirements. Otherwise, correct labels, logical page structure, logical focus order, and make alt text input for UGC images easy and prioritized will all be a great start.

https://damus.io/note15n62vjahkxz4u97a59yskqn9g4q8xrdlnz3ahapz3wrsnfa7nx5sc8avmg

**Comment by alltheseas** [comment]
Thank you for the detailed feedback. Shared with dev team.

**Comment by alltheseas** [comment]
> I did a quick test for [Global Accessibility Awareness Day](https://github.com/BitcoinDesign/Meta/issues/492) and found a lot of issues, even just on the profile screen. It's pretty much all labels, along with some grouping. It might seem like a lot, but this stuff is pretty straightforward to fix.
> 
> <img alt="damus-accessibility-test-230518" width="1172" src="https://user-images.githubusercontent.com/695901/239266128-6eaa047d-1364-4b52-b5a6-4a4e417aa827.png">

@BenGWeeks this is the accessibility labels work  @GBKS mentioned  during today's Damus devs call. Want to take this one 🙏 ?

**Comment by BenGWeeks** [comment]
Yeah, I'll take this one.

**Comment by GBKS** [comment]
If you'd like more info on testing this stuff, we have resources on the [Bitcoin Design Guide accessibility page](https://bitcoin.design/guide/designing-products/accessibility/). Here's [a short video](https://www.youtube.com/watch?v=n9FvI5SfpoU) on I usually test things.

**Comment by BenGWeeks** [comment]
Did you use the "Accessibility Inspector" in XCode? Using Audit does not finish the audit for me ...

**Comment by GBKS** [comment]
No, I just grab my phone, enable VoiceControl and VoiceOver from the accessibility short-cut menu (like in the linked video) and tap around the app. Basically testing what the end-users have in their hands.

**Comment by alltheseas** [comment]
Related https://github.com/damus-io/damus/issues/1297

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
